pkg/info/info.go

```diff
@@ -36,19 +44,23 @@ func getEksctlVersion() string {
 
 // getKubectlVersion returns the kubectl version
 func getKubectlVersion() string {
-	cmd := exec.Command("kubectl", "version", "--short", "--client")
+	cmd := exec.Command("kubectl", "version", "--client", "--output", "json")
 	out, err := cmd.CombinedOutput()
 	if err != nil {
 		return fmt.Sprintf("error : %v", err)
 	}
 
-	values := strings.FieldsFunc(string(out), func(c rune) bool {
-		return unicode.IsSpace(c)
-	})
-	if len(values) == 0 {
+	var info kubectlInfo
+
+	if err := json.Unmarshal([]byte(string(out)), &info); err != nil {
+		return fmt.Sprintf("error : %v", err)
+	}
+
+	if len(info.ClientVersion.GitVersion) == 0 {
```

Comment:
`GitVersion` is a string, it can't be compared with an integer. I think you meant:

```suggestion
	if len(info.ClientVersion.GitVersion) == "" {
```

Reply:
@cPu1  that's a `len` compare there. :)

Reply:
Ah, I didn't see that.